### DISCUSSION TITLE
Remote USB/Serial access at UPS ?

### DISCUSSION THREAD

**Discussion by vincegre** [hassio-addons/app-nut#254] [General]
Hi

I have a bunch of Eaton 3S UPS but they are spread around house ! Is there a way to get all serial/usb ports transported through network and get them "virtually" connected at add-on ? I was thinking like using some ESP for example to make a bridge for serial port to network but not sure it's possible to get it working with addon ?

Thanks

Vincèn

**Comment by sinclairpaul** [comment]
The addon utilises the supervisor for the hardware (i.e. USB/Serial Ports) so the support would need to come from there.

Apart from having them in NUT for the integration to work, you would likely run into difficulties anyway in Shutdown situations having them centrally connected.

**Reply by vincegre** [reply]
Well the goal is mainly to monitor UPS for failure in term of hardware or battery not really when it's on battery ;) I'll have a look if something can be done with supervisor but I'm not very confident :/
Thanks anyway :+1:
